examples/e2e_counter/reader.go

```diff
@@ -19,23 +20,27 @@ func runReaderPeer(repo string) {
 	checkErr(err)
 	defer n.Close()
 
-	d, err := db.NewDBFromAddr(context.Background(), n, writerAddr, key, db.WithRepoPath(repo))
+	cc := db.CollectionConfig{
+		Name:   "counter",
+		Schema: util.SchemaFromInstance(&myCounter{}),
+	}
+
+	d, err := db.NewDBFromAddr(context.Background(), n, writerAddr, key, db.WithRepoPath(repo), db.WithCollections(cc))
```

Comment:
Heres an example of creating a db from invite and passing in a collection config so the auto start works nicely

Reply:
Feels good 💯 💯 💯

Reply:
Yep!